tasks/tls_signed.yml

```diff
@@ -1,32 +1,53 @@
 ---
-- name: "[SIGNED TLS] - Define signed certificate path"
+- name: "[SIGNED TLS] - Define signed certificate-file with abolute path"
   ansible.builtin.set_fact:
-    nextcloud_tls_cert_file: "{{ nextcloud_tls_cert | default(\"/etc/ssl/\" + nextcloud_instance_name + \".crt\") }}"
+    nextcloud_tls_cert_file: "{{ nextcloud_tls_cert | default(cert_path + nextcloud_instance_name + \".crt\") }}"
 
-- name: "[SIGNED TLS] - Define signed certificate's key path"
+- name: "[SIGNED TLS] - Define signed certificate's key-file with absolute path"
   ansible.builtin.set_fact:
-    nextcloud_tls_cert_key_file: "{{ nextcloud_tls_cert_key | default(\"/etc/ssl/\" + nextcloud_instance_name + \".key\") }}"
+    nextcloud_tls_cert_key_file: "{{ nextcloud_tls_cert_key | default(cert_path + nextcloud_instance_name + \".key\") }}"
```

Comment:
While checking changes in README I've noticed that: `nextcloud_tls_cert_key` is mentioned in README only in `installed` option. So maybe there is documentation missing or it's a typo here. Could you check @marioqxx?

Reply:
@Wiktor: This seems to overlap with @staticdev's comment. I have updated the README and double-checked. So I hope I have not looked over something further.

Reply:
In this line there is:
    nextcloud_tls_cert_key_file: "{{ **nextcloud_tls_cert_key** | default(cert_path + nextcloud_instance_name + \".key\") }}"
which is not defined in **signed** section in README.

Reply:
Also resolved. See my former comment.